src/scss/components/dropdown-select/mixins/SelectControl.scss

```diff
@@ -0,0 +1,63 @@
+// Mixins to style a native <select> control.
+
+@mixin __selectControl($focused: false) {
+  @if $focused {
+    &:-moz-reset-focusring {
+      background-color: $gray-900;
+    }
+
+    outline: none;
+  }
+  @else {
+    background-color: $gray-0;
+    color: $gray-900;
+  }
+}
+
+@mixin __selectControl--invalid($focused: false) {
+  @if $focused {
+    &:-moz-focusring {
+      color: transparent;
+      text-shadow: 0 0 0 $gray-900;
+    }
+
+    outline: none;
+  }
+  @else {
+    background-color: $gray-0;
+    color: $gray-900;
+  }
+}
+
+@mixin __selectControl--disabled($focused: false) {
+  @if $focused {
+    &:-moz-reset-focusring {
```

Comment:
Incorrect pseudo element. Please change to `&:-moz-focusring`

Reply:
Changes have been done, please review.